src/main/java/io/divolte/server/SchemaRegistry.java

```diff
@@ -29,14 +29,26 @@ public SchemaRegistry(final ValidatedConfiguration vc) {
         final ImmutableMap<String,Optional<String>> schemaLocationsByMapping =
                 ImmutableMap.copyOf(Maps.transformValues(mappings, config -> config.schemaFile));
 
+        // Build a mapping of the schema location for each mapping.
+        final ImmutableMap<String,Optional<Integer>> schemaIdsByLocation =
```

Comment:
this mapping is only used for constructing the by-mapping and by-sinkname maps.  It is an intermediate result.  Alternative would be to create a separate SchemaConfiguration class, but that would be cumbersome to explain to Jackson or would impact the configuration format.  Neither seemed justified for this use.